### PR TITLE
Feature battlefields responsive

### PR DESCRIPTION
**Pull request by sirhazman07**
This Feature add 2 more html pages:
- Bus 
- Restaurant
**UI Changes:**
-New **JPG** assets for 2 new pages
- added popper.min.js script to each page to enable tooltip and minor inline JavaScript to enable tooltip next to the header title
-fixed image refs pointing to the correct asset path
- css class for tooltip icon centering
- responsive Chrome pages (not tested on Firefox or IE)

### CODE REVIEW

File: app/public/covid.html [hunk 1]
```diff
@@ -41,7 +43,7 @@
           <div class="col-sm-10 col-md-4 col-lg-4">
             <div class="thumbnail">
               <!-- <a href="/w3images/lights.jpg" target="_blank"> -->
-                <img src="/assets/52683-18636_edited_2.jpg" alt="Lights" style="width:100%">
+                <img src="../public/assets/52683-18636_edited_2.jpg" alt="Lights" style="width:100%">
```
Comment (ale-sanchez-g) [inline app/public/covid.html:46]: remove ../public

File: app/public/bus.html [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        </p>
+        </div>
+    </div>
+
+    <div class="row">
+        <div class="col-sm-12 col-md-offset-1 d-none d-md-block col-md-10 col col-lg-10">
+            <div class="thumbnail rounded mx-auto d-block">
+                <div class="col-lg">
+                    <img src="">
+                    <img class="map-height img-responsive" src="../public/assets/24745_cropped.jpg" alt="Public bus background">
```
[the remark this answers is not in the document]
Comment (sirhazman07) [reply app/public/bus.html:27]: changed src to : 
`assets/24745_cropped.jpg`

File: app/public/covid.html [hunk 3]
```diff
@@ -21,7 +23,7 @@
             <div class="thumbnail rounded mx-auto d-block">
                 <div class="col-lg">
                     <img src="">
-                    <img class="map-height img-responsive" src="/assets/2983550_cropped.jpg" alt="World map">
+                    <img class="map-height img-responsive" src="../public/assets/2983550_cropped.jpg" alt="World map">
```
[the remark this answers is not in the document]
Comment (sirhazman07) [reply app/public/covid.html:26]: same change here

File: app/public/covid.html (hunk 1 above)
Comment (sirhazman07) [reply app/public/covid.html:46]: and here

**Pull request closed without merging**